vscode/install.bash

```diff
@@ -1,8 +1,27 @@
 #! /usr/bin/env bash
 
-if [ ! -f /etc/apt/sources.list.d/vscode.list ]
+tue-install-snap-now code
+
+tue-install-system-now default-jre
+
+# Install ros extension
+code --install-extension ms-iot.vscode-ros
+
+# Configure workspace
+workspacefile="${TUE_SYSTEM_DIR}/.vscode/settings.json"
+mkdir -p "${TUE_SYSTEM_DIR}/.vscode"
+if [[ ! -f "${workspacefile}" ]]
+then
+    tue-install-cp workspace_settings.json "${workspacefile}"
+else
+    tue-install-system-now jq
+    jq -s add "${workspacefile}" "${TUE_INSTALL_CURRENT_TARGET_DIR}/workspace_settings.json" > "${workspacefile}.new"
+    $? && mv "${workspacefile}.new" "${workspacefile}"  # Doesn't work at once above
+fi
+
+# Install and configure catkin tools extension
+code --install-extension betwo.b2-catkin-tools
+if ! catkin config --workspace "$TUE_SYSTEM_DIR" | grep -q "DCMAKE_EXPORT_COMPILE_COMMANDS"
 then
-	wget "https://go.microsoft.com/fwlink/?LinkID=760868" -O /tmp/vscode.deb
-	sudo dpkg -i /tmp/vscode.deb
-	sudo apt-get install --assume-yes -f # Install dependencies
+    catkin config --workspace "$TUE_SYSTEM_DIR" --append-args -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
```

Comment:
@MatthijsBurgh I think we can add the `-DCMAKE_EXPORT_COMPILE_COMMANDS` into `tue-make` it is handy with almost all editors that have clang-format support enabled.

Reply:
I really have no idea what `-DCMAKE_EXPORT_COMPILE_COMMANDS` does. If you think we should, create a PR and let CI prove it for a cpp package.

Reply:
I'll do a local test first as I always need compile commands with vim for linting to work properly. Then create a PR if needed. Until then I'll keep this merge on hold.

Reply:
But why do that and not configure it for every ide that needs this?
Like we do for all dependencies. It's more explicit so people know why this setting is set on their setup. 

And the robots might not need it themselves..

Reply:
That's a valid point.

I see another issue though. This target should depend on `ros` so that catkin configuration doesn't get overwritten. Also it will be better to add a `setup` file that has the configuration part so that when we clean up the ros target, the target remains unaffected.

Reply:
I was just thinking of creating a `ros-setup` target. On which the `ros` and `catkin` target can depend.